docs/usage.md

```diff
@@ -0,0 +1,86 @@
+# Usage
+
+Use the import tool from the command line, like so:
+```
+java -jar <path-to-jar> <mode> <list of mcf/tmcf/csv files>
+```
+
+Hint: it can be useful to create an alias for the jar file, such as:
```

Comment:
Perhaps use blockquote or notes? https://about.gitlab.com/handbook/markdown-guide/#blockquotes

Reply:
+

Reply:
Was this done?